cost-analyzer/values.yaml

```diff
@@ -1290,6 +1290,12 @@ costEventsAudit:
 #    daemonset_external_label: "kubernetes_daemonset"
 #    pod_external_label: "kubernetes_pod"
 #  grafanaURL: ""
+#  # Provide a mapping from Account ID to a readable account name which will be displayed in the UI for CloudCosts
+#  # Account ID refers to different entities for different entities for each provider:
+#  # - AWS: Account ID
+#  # - Azure: Subscription ID
+#  # - GCP: Project ID
+#  cloudAccountMapping: {}
```

Comment:
Worth adding an example for users here? It's not entirely clear what format I should pass to this newb user.

Reply:
Agreed. The comment structure is also confusing here.